time/test/time.ts

```diff
@@ -50,6 +50,41 @@ function map(stream: any, f: any) {
   return stream.map(f);
 }
 
+function testUnsubscription(Time: TimeSource, operator: any, done: Mocha.Done) {
+  const PERIOD = 20;
+  const taps: Array<number> = []; // tslint:disable-line
```

Comment:
why is the tslint disabled?

Reply:
oh yeah it's not necessary here anymore
I had declared 
`let taps: Array<number> = [];`  but then tslint complained : 
`Identifier 'taps' is never reassigned; use 'const' instead of 'let'.`

Reply:
Updated :)